paddle/fluid/operators/mkldnn/fc_mkldnn_op.cc

```diff
@@ -166,6 +170,15 @@ class FCMKLDNNHandler
       }
     }
 
+    if (ctx.HasAttr("fuse_residual_connection") &&
+        ctx.Attr<bool>("fuse_residual_connection")) {
+      // For Inner Product primitives, the destination always N * C
+      auto residual_data_md = dnnl::memory::desc(
+          {MB, OC}, dnnl::memory::data_type::f32, dnnl::memory::format_tag::ab);
```

Comment:
recommend to get_mdesc from the residual input tensor instead of assumption here.

Reply:
For inner product primitive, the dst is always N*C(mentioned in doc). That's why we can not use residual mdesc directly, have to make sure residual md shape is N*C.

Reply:
when the residual won't be NC? BTW, the data type can also be bf16, right?